### DISCUSSION TITLE
Raw text files (headers) as Inputs to DemoModel() Class

### DISCUSSION THREAD

**Discussion by solidkon** [mupif/mupif#57] [Q&A] [answered]
Hi!

Can you please provide an example of how to define a raw text file (header) as an input for the MD of the DemoModel() Class? 
What I mean by a header text file:

This would be a templated input file needed for an application where some fields may be modified by the API itself but others will not (at least for the moment).

**Comment by stanislavsulc** [comment, marked as answer]
Hi, an example of passing a text file into a model is here: https://github.com/mupif/mupif/blob/6d4462d5f07a045568e5b030073fb0bd71aa9409/examples/07-stacTM-JobMan-distrib/Example07.py#L96-L98
The model has information about this input in the metadata, see https://github.com/mupif/mupif/blob/6d4462d5f07a045568e5b030073fb0bd71aa9409/examples/models.py#L56-L64
A workflow can get a file as an input, it can do any changes with it and then pass it into a model analogically.
